qoqo/src/noise_models/continuous_decoherence.rs

```diff
@@ -56,10 +56,16 @@ impl ContinuousDecoherenceModelWrapper {
     #[new]
     pub fn new(noise_operator: Option<&Bound<PyAny>>) -> PyResult<Self> {
         if let Some(lindblad_operator) = noise_operator {
-            let noise_operator =
-                struqture_py::spins::PlusMinusLindbladNoiseOperatorWrapper::from_pyany(
+            let noise_operator: struqture::spins::PlusMinusLindbladNoiseOperator =
+                match struqture_py::spins::PlusMinusLindbladNoiseOperatorWrapper::from_pyany(
                     lindblad_operator,
-                )?;
+                ) {
+                    Ok(x) => x,
+                    Err(_) => match struqture_py::spins::PlusMinusLindbladNoiseOperatorWrapper::from_struqture_2(lindblad_operator) {
+                        Ok(x) => x.internal,
```

Comment:
More of a struqture question: why does the first `Ok` lead to x but the second one to `x.internal`? A 2.0 refactor?

Reply:
The first one returns the internal object, but the second function returns a wrapper - this is a good point, I'll add this to the list of things to review for struqture